packages/gatsby-plugin-sass/src/gatsby-node.js

```diff
@@ -2,12 +2,16 @@ import resolve from "./resolve"
 
 exports.onCreateWebpackConfig = (
   { actions, stage, rules, plugins, loaders },
-  { cssLoaderOptions = {}, postCssPlugins, ...sassOptions }
+  { cssLoaderOptions = {}, postCssPlugins, implementation, ...sassOptions }
 ) => {
   const { setWebpackConfig } = actions
   const PRODUCTION = stage !== `develop`
   const isSSR = stage.includes(`html`)
 
+  if (implementation && implementation === `dart-sass`) {
+    sassOptions.implementation = require(`sass`)
```

Comment:
We should do some sanity checks here - if user didn't install `sass` package, we probably should display warning - something like this :
```suggestion
    try {
      sassOptions.implementation = require(`sass`)
    } catch (e) {
      reporter.warn(`"sass" package isn't installed. Reverting to "node-sass".`)
    }
```
(`reporter` is passed as field in first arg, so you would need to destructure `reporter` in line 4)

Reply:
will update now. I added that early but node.js by default will throw module not found exception and hence i ignored it.

Reply:
> Reverting to "node-sass". 

If we say so then, `node-sass` should be in dependency. 
Also, Instead of warning. We should throw errors. Since No package is installed build will break